components/style/values/computed/svg.rs

```diff
@@ -34,17 +37,75 @@ impl SVGPaint {
     }
 }
 
+/// <length> | <number> | <percentage>
+/// This valule is for SVG values which allow the unitless length.
+#[allow(missing_docs)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[derive(Clone, Debug, PartialEq, ToCss)]
+pub enum SVGLengthOrPercentageOrNumber {
```

Comment:
Nit: could you name it `SvgLengthOrPercentage`? The `Number` variant is just a unitless length right?